llvm/test/Transforms/LICM/update-scev-after-hoist.ll

```diff
@@ -1,13 +1,33 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 5
```

Comment:
Can you please pre-commit the regeneration of the check lines? Hard to see what actually changed here.